README.md

```diff
@@ -1,42 +1,89 @@
+[![InstantSearch.js logo][logo]][website]
+
+InstantSearch.js is a library for building blazing fast search-as-you-type search UIs with [Algolia](algolia-website).
+
+InstantSearch.js is part of the InstantSearch family: 
+**InstantSearch.js** 
+| [React InstantSearch (web)][react-instantsearch-github]
+| [Vue InstantSearch (web)][vue-instantsearch-github]
+| [InstantSearch Android (native)][instantsearch-android-github]
+| [InstantSearch iOS (native)][instantsearch-ios-github].
+
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-- [InstantSearch.js](#instantsearchjs)
-- [Documentation](#documentation)
-- [Support](#support)
-- [Contributing, dev, release](#contributing-dev-release)
+
+- [About InstantSearch.js ⚡️](#about-instantsearchjs-%EF%B8%8F)
+- [Getting started ⌨️](#getting-started-)
+- [Browser support 🌍](#browser-support-)
+- [Contributing, dev, release 🚀](#contributing-dev-release-)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-[![InstantSearch.js logo][logo]][website]
+## About InstantSearch.js ⚡️
+
+> ⚡ Lightning-fast search for your apps
 
-InstantSearch family: **InstantSearch.js** | [React InstantSearch][react-instantsearch-github] | [InstantSearch Android][instantsearch-android-github] | [InstantSearch iOS][instantsearch-ios-github].
+This library is built and maintained by [Algolia][algolia-website].
```

Comment:
Since we want to encourage contributors, maybe we should change this sentence to something like:

> This library is built and maintained by contributors and Algolians